charters/wot-wg-charter-draft-2019.html

```diff
@@ -306,160 +692,251 @@ <h2>Success Criteria</h2>
+            <dd>For collaboration on testbeds and on gathering user requirements for the Web of Things.</dd>
+            <dt><a href="https://opcfoundation.org">OPC Foundation</a></dt>
+            <dd>For collaboration on semantic interoperability in terms of information model, protocol binding, and security.</dd>
+            <dt><a href="http://www.plattform-i40.de/I40/Navigation/EN/Home/home.html">Plattform Industrie 4.0</a></dt>
+            <dd>For collaboration on use cases and requirements for smart manufacturing, including semantic interoperability and end to end security across platforms.</dd>
+            <dt><a href="http://www.onem2m.org">oneM2M</a></dt>
+            <dd>For collaboration on integrating M2M based platforms within the Web of Things, including platform metadata and approaches for enabling semantic interoperability, and end to end security across platforms.</dd>
+            <dt><a href="https://www.etsi.org">ETSI</a></dt>
+            <dd>For collaboration on IoT, M2M, security, and semantic interoperability.</dd>
+            <dt><a href="https://www.omaspecworks.org">OMA SpecWorks</a></dt>
+            <dd>For collaboration on integrating LWM2M based platforms with the Web of Things.</dd>
+            <dt><a href="http://www.opengeospatial.org/">Open Geospatial Consortium</a></dt>
+            <dd>For coordination on the OGC SensorThings API and location metadata.</dd>
+            <dt><a href="http://www.gsma.com">GSMA</a></dt>
+            <dd>For coordination in respect to the interests of Network Operators.</dd>
+            <dt><a href="https://www.eclass.eu/en.html">eCl@ss</a></dt>
+            <dd>For collaboration and coordination of the technical realization of the use of eCl@ss in the W3C Thing Description.</dd>
+            <dt><a href="https://aioti.eu/">AIOTI WG03</a></dt>
+            <dd>For coordination with the Alliance for Internet of Things Innovation, Working Group 3 (standardisation), in respect to use cases, requirements, and semantic interoperability.</dd>
           </dl>
         </section>
       </section>
 
-
-
       <section class="participation">
         <h2 id="participation">
           Participation
         </h2>
         <p>
-          To be successful, this <i class="todo">(Working|Interest)</i> Group is expected to have 6 or more active participants for its duration, including representatives from the key implementors of this specification, and active Editors and Test Leads for each specification. The Chairs, specification Editors, and Test Leads are expected to contribute half of a working day per week towards the (Working|Interest) Group. There is no minimum requirement for other Participants.
+          To be successful, 
+          this Working Group is expected to have 6 or more active participants for its duration, 
+          including representatives from the key implementors of this specification, 
```

Comment:
Ideally the WG would be balanced across different constituencies, with members from
- home end-device manufacturers
- industrial end-device manufacturers
- gateway/edge device manufacturers
- cloud providers

Reply:
OK, let me see if I can reword this to make it more balanced and to list all the stakeholders.

Reply:
I added a sentence about "balanced" similar to above, except I generalized the end-device manufacturers to say "various verticals".